### DISCUSSION TITLE
Web interface localhost:9000 not working/ redirects to console address port using latest minio.exe on Windows 10

### DISCUSSION THREAD

**Discussion by dwolfschlaeger** [minio/minio#12839] [Q&A]
## Expected Behavior
In previous versions, starting the minio.exe server via Windows Cmd/ Powershell and accessing localhost:9000 made it possible to enter the credentials and use the web interface to create bucket etc.

## Current Behavior
accessing localhost:9000  redirects to localhost:<specific or dynamic> and keeps loading forever

## Steps to Reproduce (for bugs)
1. Download: https://dl.min.io/server/minio/release/windows-amd64/minio.exe
2. Open CMD
3. Enter: minio.exe server D:\test
4. Open browser and enter: localhost:9000

## Your Environment
I use the minio server for local development in Windows 10.
minio release date 2021-07-30

**Comment by dwolfschlaeger** [comment]
Is anyone epxeriencing the same? Until now, I did not have any problems using the minio server in the described manner. The problem occured when I was helping a student I supervise setting up his development environment for the first time on his notebook. 
I appreciate your help in advance :)

**Comment by vadmeste** [comment]
> accessing localhost:9000 redirects to localhost: and keeps loading forever

Is it litterally redirecting to `localhost:` without any port ? 

Can you start using `--console-address ":9001"` and try to open http://localhost:9001 ?

**Comment by madone1** [comment]
I just noticed the same thing a couple of days ago.  If I start any of the versions on windows 10 since they split out the ports, the login box never appears.  It just keeps loading.  I can specify a port and it behaves the same way.  If I download the Linux version, I can run it under WSL2 and it works fine.

**Reply by klauspost** [reply]
Looked into it a month or so ago. On my system it picks up the wrong IP to redirect to, leading it to redirect to an unavailable IP.

**Comment by mmm8955405** [comment]
In CentOS 8 system, I encountered the same problem
